test/conftest.py

```diff
@@ -1,5 +1,8 @@
+import os
+
 from numpyro.util import set_rng_seed
 
 
 def pytest_runtest_setup(item):
     set_rng_seed(0)
+    os.environ['XLA_FLAGS'] = '--xla_cpu_enable_fast_math=false'
```

Comment:
+1 nice!

Reply:
This doesn't work actually, so I have removed it. The problem is that it needs to be set much earlier, so we will need to set the environment variable manually. I had to do this because the NUTS tests wouldn't terminate otherwise.